jans-cedarling/bindings/cedarling_python/src/config/authz_config.rs

```diff
@@ -0,0 +1,82 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use pyo3::exceptions::PyValueError;
+use pyo3::prelude::*;
+
+/// AuthzConfig
```

Comment:
The comments in these wrapper files are way too big. Please use shorter comments. If you want to document python code, please use https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings styles for comment styles

Reply:
I tried to make documentation concise and to follow the style guide you provided. Also used ChatGPT.  So hope new doc strings is better.